src/figuresClasses.ts

```diff
@@ -1,19 +1,91 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
+type Color = 'red'|'green'|'blue';
+type Shape = 'triangle'|'circle'|'rectangle';
+
 export class Triangle {
+  public shape: Shape;
+
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.shape = 'triangle';
+
+    const first : boolean = this.a + this.b > this.c;
+    const second : boolean = this.a + this.c > this.b;
+    const third : boolean = this.b + this.c > this.a;
```

Comment:
You don't need to define types for primitive const variables.
```suggestion
    const first = this.a + this.b >= this.c;
    const second = this.a + this.c >= this.b;
    const third = this.b + this.c >= this.a;
```